tests/fixtures/manifest-v12.json

```diff
@@ -36,7 +36,9 @@
                 "schema": null,
                 "database": null,
                 "tags": [],
-                "meta": {},
+                "meta": {
```

Comment:
This file gets overwritten from the sandbox dbt project, easiest thing would be to put these in one of the older version manifests that don't get overwritten

Reply:
Can I just change the `sandbox` content definition to match this file then? And update older versions as well, so that they all match

Reply:
You could, but that might involve debugging other tests if the manifest has other updates, it would be easier if you just updated one older version of the manifest (and corresponding assertions). They don't all need to match, one is sufficient to test this functionality somewhere